webflow-hacks/form-hacks/29/29.ts

```diff
@@ -0,0 +1,22 @@
+// when DOM is ready
+document.addEventListener('DOMContentLoaded', function () {
+  // declare constant selectors
+  const FORM_SELECTOR = '[fs-hacks-element="hack29-form"]';
+  const NAME_INPUT_SELECTOR = '[fs-hacks-element="hack29-name-input"]';
+  const MESSAGE_SELECTOR = '.w-form-done';
+  const messageDiv = document.querySelector<HTMLDivElement>(MESSAGE_SELECTOR);
```

Comment:
The `messageDiv` should be also queried under the form scope, as ther emight be more than 1 form on the page.